solutions/usaco-664.mdx

```diff
@@ -83,4 +83,52 @@ for i in ans:
 
 </PySection>
 
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class BlockGame {
+
+    /**
+     * Method counts every occurance of characters in a word.
+     * It returns the array containing letter frequency.
+     */
```

Comment:
indentation is screwed up again, use tab here

Reply:
Yes, I have seen this. I'm still testing everything. I will call you for rereview :))

Reply:
Oh, I resolved it changes. Totally my bad.